LOAN/LoansSpecific/CardAccounts.rdf

```diff
@@ -321,6 +363,9 @@
 		</rdfs:subClassOf>
 		<rdfs:label>credit card</rdfs:label>
 		<skos:definition>card issued by a financial service provider that enables the cardholder to borrow funds</skos:definition>
+		<cmns-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.fdic.gov/regulations/examinations/credit_card/pdf_version/ch2.pdf</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote>In its non-physical form, a credit card represents a payment mechanism which facilitates both consumer and commercial business transactions, including purchases and cash advances. A credit card generally operates as a substitute for cash or a check and most often provides an unsecured revolving line of credit. The borrower is required to pay at least part of the card&apos;s outstanding balance each billing cycle, depending on the terms as set forth in the cardholder agreement. As the debt reduces, the available credit increases for accounts in good standing. These complex financial arrangements have ever-shifting terms and prices. A charge card differs from a credit card in that the charge card must be paid in full each month.</cmns-av:explanatoryNote>
+		<cmns-av:explanatoryNote>In physical form, a credit card traditionally is a thin, rectangular plastic card. The front of the card contains a series of numbers that are representative of various items such as the applicable network, bank, and account.</cmns-av:explanatoryNote>
```

Comment:
I have at least one credit card where the credit card number is on the back of the card

Reply:
There are also now virtual cards for online payments, for the same account, but with a different card number. Can we model those?